monai/networks/nets/netadapter.py

```diff
@@ -71,33 +71,42 @@ def __init__(
             in_channels_ = in_channels
 
         if pool is None:
-            # remove the last layer
-            self.features = torch.nn.Sequential(*layers[:-1])
+            # remove the last layer or replace it with an identity
+            if hasattr(model, "fc"):  # assuming fc is the last layer
```

Comment:
Is there any value in making the name a constructor argument rather than hard-coding "fc", or is this class used only in situations that would follow this pattern?